library_generation/.gitignore

```diff
@@ -0,0 +1,8 @@
+gapic-generator-java*
+google/
+grpc-java-plugin*
+protobuf*
+output/
+schema/
+out
+protoc*
```

Comment:
This seems like a very broad filter that can accidentally ignore needed files. Can we just use a single working directory that would be excluded or do proper cleanup at the end of the generation process?

Reply:
I removed the .gitignore file and addded a convenience function to perform cleanup